.github/workflows/tkg_integration_tests_clusterclass.yaml

```
@@ -40,6 +40,87 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  integ-tests-docker-cc:
+    name: TKG Integration Tests CAPD CC
```

We should use consistent naming here similar to what we are using for CAPA?


```suggestion
    name: TKG Integration Tests CAPD (ClusterClass)
```